### PR TITLE
chore: flutter session replay onboarding

### PR DESCRIPTION
**Pull request by marandaneto**
## Changes

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

File: frontend/src/scenes/onboarding/sdks/product-analytics/flutter.tsx [hunk 1]
```diff
@@ -18,6 +22,8 @@ export function ProductAnalyticsFlutterInstructions(): JSX.Element {
             <SDKInstallFlutterInstructions />
             <h3>Send an Event</h3>
             <FlutterCaptureSnippet />
+            <PersonModeEventPropertyInstructions />
```
Comment (marandaneto) [inline frontend/src/scenes/onboarding/sdks/product-analytics/flutter.tsx:25]: fly by fix since flutter has person mode

**Review by ioannisj — APPROVED** [review]
Not 100% sure what I'm seeing but 👍

**Review by pauldambra — APPROVED** [review]
🚀

**Review by pauldambra — APPROVED** [review]
🚀

**Comment by marandaneto** [comment]
> Not 100% sure what I'm seeing but 👍

if you create a fresh org or project, you need to select a platform where you install sdks, and those are the onboarding steps for those cases, for example https://us.posthog.com/project/76102/onboarding/feature_flags?step=install

**Pull request merged**